Content.Server/Radio/Components/HandheldRadioComponent.cs

```diff
@@ -1,101 +1,74 @@
-using Content.Server.Chat;
-using Content.Server.Chat.Systems;
 using Content.Server.Radio.EntitySystems;
+using Content.Server.RadioKey.Components;
 using Content.Shared.Interaction;
-using Content.Shared.Popups;
 using Content.Shared.Radio;
-using Robust.Shared.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
 
 namespace Content.Server.Radio.Components
 {
     [RegisterComponent]
     [ComponentProtoName("Radio")]
     [ComponentReference(typeof(IRadio))]
     [ComponentReference(typeof(IListen))]
-#pragma warning disable 618
     public sealed class HandheldRadioComponent : Component, IListen, IRadio
```

Comment:
If you're refactoring radio you should really kill IListen and IRadio in the process.